dandi/validate.py

```diff
@@ -42,7 +41,7 @@ def validate_file(filepath):
 def validate_dandiset_yaml(filepath):
     """Validate dandiset.yaml"""
     with open(filepath) as f:
-        meta = yaml.safe_load(f)
+        meta = yaml_load(f, typ="safe")
```

Comment:
just a mental note unrelated to this PR -- apparently we nohow/nowhere validate `dandiset.yaml`? will be handled properly anyways soon with RFing of metadata and validation